client/src/locales/en.ts

```diff
@@ -326,6 +326,14 @@ export default {
 			"Administrators are granted everything. Room owner is automatically Administrator, and can't be demoted.",
 		"viewing-as": "Viewing as",
 		"permission": "Permission",
+		"roles": {
+			"administrator": "Administrator",
```

Comment:
You need to use the `Role` enum as the keys for this.

Ctrl + f this file for "OttWebsocketError" to see what I'm talking about.